json4s/src/test/scala/org/http4s/json4s/Json4sSpec.scala

```diff
@@ -1,14 +1,14 @@
 package org.http4s
 package json4s
 
+import cats.syntax.option._
```

Comment:
Ah yeah, I saw that somewhere else, just didn't connect the dots. Will keep that in mind.